slam-libraries/viam-cartographer/src/slam_service/config.cc

```diff
@@ -30,6 +30,12 @@ DEFINE_int64(
     "Frequency at which we want to print map pictures while cartographer "
     "is running.");
 DEFINE_string(input_file_pattern, "", "Input file pattern");
+// TODO: Switch back to bool once PR #1689 is submitted
```

Comment:
Is the plan to merge this PR, followed by #1689, followed by a PR to change this to a bool (which makes it required)?

Reply:
Yes I've added JIRA Ticket: https://viam.atlassian.net/browse/RSDK-1593 to cover this once other PRs are merged